src/cGame_logic.cpp

```diff
@@ -716,6 +716,7 @@ void cGame::shakeScreen(int duration)
 */
 void cGame::shutdown()
 {
+    cParticle::reset();
```

Comment:
We would need to reset cache upon new mission init I think? Because there is no guarentee the same player index have the same colors in the next (skirmish) mission.

Reply:
Sure! 100% right. GG on you.

Reply:
Lets resolve this conversation when we fix this